### PR TITLE
add to docs index page

### PR DESCRIPTION
**Pull request by CrowdHailer**

### CODE REVIEW

File: docs/invitations/index.md [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+Magic invitations frictionlessly authenticate your users and allow them to make use of your application immediately.
+They can be sent to:
+
+- Add a colleague to review your documents
+- Share a photo with family members
+- Invite a new user to start using your service.
+
+## Steps
+
+1. Create a template for you invitation email.
```
Comment (CrowdHailer) [inline docs/invitations/index.md:18]: This is the step that is not the easiest to document. I think it's worth creating an invite email which say Alice invited Bob to service X

File: docs/index.md [hunk 2]
```diff
@@ -11,11 +11,24 @@ sitemap:
...
+- Share a photo with family members
+- Invite a new user to start using your service.
+
+All of these use cases can be achieved using DID.app, follow our [guide for sending secure invitations](/docs/invitations).
+
+### Authenticate with Magic Links
+
+Magic invitations are a great way to grow your service,
+and can be used next to your existing authentication solution.
+However, you do not need any other authentication as users can sign-in using magic links send by DID.app.
```
Comment (richardesigns) [inline docs/index.md:29]: `[](url)`
```suggestion
However, you do not need any other authentication as users can sign-in using magic links sent by DID.app.
```

File: docs/index.md [hunk 3]
```diff
@@ -11,11 +11,24 @@ sitemap:
...
+
+To set up authentication follow our step-by-step [integration guide](/docs/step-by-step-integration).
```
Comment (richardesigns) [inline docs/index.md:31]:
```suggestion
To set up authentication, follow our step-by-step [integration guide](/docs/step-by-step-integration).
```

File: docs/invitations/index.md [hunk 4]
```diff
@@ -1,65 +1,92 @@
...
-  -d login_hint=new_user@example.test \
-  -d response_mode=fragment \
-  -d template_alias=user-invitation \
-  -d data_inviter_name=Sarah
+```
+https://myapp.com/welcome#code=CODE
 ```
 
-### Handling Authorization Responses
+Fetch a users information from the token endpoint using the code.
```
Comment (richardesigns) [inline docs/invitations/index.md:61]:
```suggestion
Fetch a user's information from the token endpoint using the code.
```

**Pull request merged**